lib/managers/collaboration-whitelists.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * @fileoverview Manager for the Box Collaboration Whitelist Resource
+ */
+
+'use strict';
+// -----------------------------------------------------------------------------
+// Typedefs
+// -----------------------------------------------------------------------------
+
+/**
+ * Collaboration Whitelist parameter constant
+ * @typedef {string} CollaborationWhitelistDirection Determines the type of restriction for whitelisting for a domain
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var urlPath = require('../util/url-path');
+
+
+// ------------------------------------------------------------------------------
+// Private
+// ------------------------------------------------------------------------------
+var BASE_PATH = '/collaboration_whitelist_entries',
+	TARGET_ENTRY_PATH = '/collaboration_whitelist_exempt_targets';
+
+// ------------------------------------------------------------------------------
+// Public
+// ------------------------------------------------------------------------------
+
+/**
+ * Simple manager for interacting with all 'Collaboration Whitelist' endpoints and actions.
+ *
+ * @constructor
+ * @param {BoxClient} client - The Box API Client that is responsible for making calls to the API
+ * @returns {void}
+ */
+function CollaborationWhitelists(client) {
```

Comment:
This should maybe be singular, since an enterprise only has one whitelist?  This is similar to what we did with trash